src/map/map.ts

```diff
@@ -335,7 +335,10 @@ export class PropertiesMap {
         if (this._is3D()) {
             data.marker.style.display = 'none';
         } else {
-            this._updateSelectedMarker(data);
+            this._active = guid;
+            const factor = this._options.size.factor.value;
+            const scaleMode = this._options.size.mode.value;
+            this._restyle({'marker.size': this._sizes(1)} as Data, 1);
```

Comment:
I don't understand why this is required in `addMarker`, could you explain?

Reply:
There's a weird bug where if I don't _restyle, any new added markers get placed in the far bottom left of the graph

Reply:
And calling `this._updateSelectedMarker(data);` do not work? I can try to give this a look once you are done with this PR